lang/errors.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Runs the specified function and returns its result or the error thrown.
+ *
+ * @param work The function to be executed
+ * @returns A `T` containing the return value of the function if it succeded, or the {@link Error} thrown if it failed
+ */
```

Comment:
Maybe I'm not familiar with Deno, but this seems like an anti pattern - calling code will have to check for existence of an Error value and handle it, which would likely be easier to do in a native try/catch?

I don't see this used within the library so I'm struggling to think of a use case for this function.

Reply:
This was another direct alternative to existing lodash functionality: https://lodash.com/docs/4.17.15#attempt

However, I think you're right that there's a more elegant way to handle this in most situations. Most places where I'm using this are for arrays of promises, but that can also be better handled by replacing calls to `Promise.all` with `Promise.allSettled`.